slides/en/index.html

```diff
@@ -465,16 +549,30 @@ <h2>Actors</h2>
             </table>
 
             <aside class="notes">
+			  8<br /><br />
+			  Explain what is a candidate and assignee<br />
+			  Actor can be related to process instantiation, single human task or lane<br />
+			  [Studio - actors list and reference to process, initiator]<br />
               Actors allow to automatically update task visibility when organization changes.<br/>
+			  Actor can be mapped to a subset of the organization using actor mapping: lists of users, groups, roles and memberships<br />
+              [Studio & Portal - edit process configuration actor mappings, run it]<br />
               Actors cannot use any live data (business variables, external system information...) whereas filters can.<br/>
               Filters are executed only once when task starts. They can be run again with API call.<br />
-              If you already have information about your organization stored in an LDAP/Active Directory server Enterprise Edition provide a tool that will save you time by dealing with the synchronization between LDAP/AD and Bonita user database.
+              [Studio & Portal - change task actor to actor filter, run it]<br />
+			  If you already have information about your organization stored in an LDAP/Active Directory server Enterprise edition provide a tool that will save you time by dealing with the synchronization between LDAP/AD and Bonita user database.<br />
+			  On the authentication purpose, Bonita provides features to use SSO (SAML V2, CAS) and LDAP. Kerberos is also something we see regularly implemented for projects.
```

Comment:
About Kerberos you can anticipate the official support coming with 7.8.0.

Reply:
Agreed